.gitignore

```diff
@@ -1,4 +1,265 @@
-dist/
-node_modules/
-*.log
+dist/
```

Comment:
Maybe we can make it add a comment or something explaining that this ignore is project specific? So we don't accidentally lose it in case we download a new one from gitignore.io?

Reply:
Great idea! While fixing this I also found that I had included from gitignore.io twice by accident.

![image](https://user-images.githubusercontent.com/2315367/54476193-beccce00-47fa-11e9-9d4f-e69a1209bd60.png)